ci/README.md

```diff
@@ -9,13 +9,68 @@ File [k8s-conformance-image-version](k8s-conformance-image-version) stores the
 version number of the K8s conformance container image we currently use to run
 tests.
 
+## Antrea test suite
+
+We run 4 different categories of tests as part of CI:
+ * **unit tests**: most Go packages for Antrea include some unit tests written
+   using the [Go `testing`] package. Unit tests typically rely on mock testing
```

Comment:
I wonder why [Go `testing`], not [`Go testing`] or Go [`testing`] ? I think It looks strange.

Reply:
Changed to Go [`testing`]